ui-components/src/navigation/ProgressNav.tsx

```diff
@@ -22,29 +26,33 @@ const ProgressNavItem = (props: {
     }
   }
 
-  const srText =
-    props.section === props.currentPageSection ? (
-      <span className="sr-only">{t("progressNav.current")}</span>
-    ) : (
-      ""
-    )
+  const srTextBuilder = (): string | React.ReactFragment => {
+    if (props.section < props.currentPageSection) {
+      return (
+        <span className="sr-only">
+          {props.strings?.screenReaderCompleted ?? t("progressNav.completed")}
+        </span>
+      )
+    } else if (props.section > props.currentPageSection) {
+      return (
+        <span className="sr-only">
+          {props.strings?.screenReaderNotCompleted ?? t("progressNav.notCompleted")}
```

Comment:
`progressNav.notCompleted` and `progressNav.completed` need to be added to the general.json file for ui-components